code/_onclick/item_attack.dm

```diff
@@ -227,6 +225,9 @@ obj/item/proc/get_clamped_volume()
 						power = K.defense(power,def_zone)
 					M.take_organ_damage(0, power)
 					to_chat(M, "Aargh it burns!")
+		. = TRUE //The attack always lands
```

Comment:
Isn't this already set at the start of the proc?

Reply:
It's set a few other places as well, notably lines 203/205 and 205/207